### PR TITLE
feat: support wechat miniprogram of qrcode && API change

### PR DESCRIPTION
**Pull request by ChrisCindy**
- support wechat miniprogram
- code refactoring for ali miniapp
- core API of `options.errorCorrectLevel`changed for eliminating differences between web/weex and miniapp

### CODE REVIEW

File: packages/rax-qrcode/package.json [hunk 1]
```diff
@@ -44,20 +45,20 @@
...
     "@rax-types/rax": "^1.0.5",
     "@types/ali-app": "^1.0.0",
     "@types/jest": "^24.0.12",
     "@typescript-eslint/eslint-plugin": "^1.7.0",
     "@typescript-eslint/parser": "^1.7.0",
     "csstype": "^2.6.4",
-    "eslint": "^5.16.0",
-    "driver-universal": "^1.0.0",
     "driver-dom": "^1.0.0",
+    "driver-universal": "^1.0.0",
```
Comment (yacheng) [inline packages/rax-qrcode/package.json]: driver-universal 3.0.0

File: packages/rax-qrcode/package.json [hunk 2]
```diff
@@ -44,20 +45,20 @@
...
     "driver-weex": "^1.0.0",
+    "eslint": "^5.16.0",
     "eslint-config-rax": "^0.0.0",
+    "miniprogram-api-typings": "^2.9.3",
     "rax-scripts": "^2.0.0",
```
Comment (yacheng) [inline packages/rax-qrcode/package.json]: 建议升级 @alib/build-scripts

File: packages/rax-qrcode/README.md [hunk 3]
```diff
@@ -12,30 +12,28 @@ $ npm install rax-qrcode --save
 ## 属性
 
 注：
-1、**支持**列表中的 <img alt="browser" src="https://gw.alicdn.com/tfs/TB1uYFobGSs3KVjSZPiXXcsiVXa-200-200.svg" width="25px" height="25px" />代表 h5 <img alt="weex" src="https://gw.alicdn.com/tfs/TB1jM0ebMaH3KVjSZFjXXcFWpXa-200-200.svg" width="25px" height="25px" />代表 weex <img alt="miniApp" src="https://gw.alicdn.com/tfs/TB1bBpmbRCw3KVjSZFuXXcAOpXa-200-200.svg" width="25px" height="25px" />代表小程序
+1、**支持**列表中的 <img alt="browser" src="https://gw.alicdn.com/tfs/TB1uYFobGSs3KVjSZPiXXcsiVXa-200-200.svg" width="25px" height="25px" />代表 h5 <img alt="weex" src="https://gw.alicdn.com/tfs/TB1jM0ebMaH3KVjSZFjXXcFWpXa-200-200.svg" width="25px" height="25px" />代表 weex <img alt="miniApp" src="https://gw.alicdn.com/tfs/TB1bBpmbRCw3KVjSZFuXXcAOpXa-200-200.svg" width="25px" height="25px" />代表阿里小程序<img alt="wechatMiniprogram" src="https://img.alicdn.com/tfs/TB1slcYdxv1gK0jSZFFXXb0sXXa-200-200.svg" width="25px" height="25px"> 代表微信小程序
```
Comment (yacheng) [inline packages/rax-qrcode/README.md]: README.md 文件用英文版， README-zh_CN.md 放中文 readme

File: packages/rax-qrcode/package.json (hunk 1 above)
Comment (ChrisCindy) [reply packages/rax-qrcode/package.json]: 已升级版本

File: packages/rax-qrcode/package.json (hunk 2 above)
Comment (ChrisCindy) [reply packages/rax-qrcode/package.json]: 如沟通，为保稳定性，暂时不升级

File: packages/rax-qrcode/README.md (hunk 3 above)
Comment (ChrisCindy) [reply packages/rax-qrcode/README.md]: 后面统一升级组件的 README

**Pull request merged**